pkg/kvstore/events.go

```diff
@@ -70,10 +70,10 @@ type stopChan chan struct{}
 // Watcher represents a KVstore watcher
 type Watcher struct {
 	// Events is the channel to which change notifications will be sent to
-	Events EventChan
+	Events EventChan `json:"-"`
 
-	name      string
-	prefix    string
+	Name      string `json:"name"`
```

Comment:
What's the reasoning behind capitalizing these fields?

Reply:
Serialization packages in go (including json) don't work unless the member is public.